sdk/helper/testcluster/replication.go

```diff
@@ -839,3 +851,33 @@ func (r *ReplicationSet) Cleanup() {
 		cluster.Cleanup()
 	}
 }
+
+func WaitForPerfReplicationConnectionStatus(ctx context.Context, client *api.Client) error {
+	return WaitForPerfReplicationStatus(ctx, client, func(m map[string]interface{}) error {
+		perfSecondaries := m["primaries"].([]interface{})
+		if len(perfSecondaries) == 0 {
+			return fmt.Errorf("secondaries is zero")
+		}
+		for _, v := range perfSecondaries {
+			if v.(map[string]interface{})["connection_status"] == "connected" {
```

Comment:
nit: would using mapstructure package be simplifying this?

Reply:
Haha it surely would be less of a panic risk.  (laughing bc I think you're throwing one of my suggestions back in my face).  I'll fix.

Reply:
Fixed, thanks for suggestion.